packages/twenty-front/src/modules/workflow/types/Workflow.ts

```diff
@@ -46,14 +48,23 @@ export type WorkflowDatabaseEventTrigger = BaseTrigger & {
 
 export type WorkflowTrigger = WorkflowDatabaseEventTrigger;
 
+export type WorkflowStatus = 'DRAFT' | 'ACTIVE' | 'DEACTIVATED';
+
+export type WorkflowVersionStatus =
+  | 'DRAFT'
+  | 'ACTIVE'
+  | 'DEACTIVATED'
+  | 'ARCHIVED';
```

Comment:
think it would be better to import `WorkflowVersionStatus` and `WorkflowStatus` from `~/generated/graphql`, for that you need to use `registerEnumType` in `workflow-version.workspace-entity.ts` and `workflow.workspace-entity.ts` like it is done in `workspace.entity.ts` for `WorkspaceActivationStatus`

Reply:
It seems that I won't be able to generate those types for now. I will define these types manually.